README.md

```diff
@@ -17,18 +17,23 @@ and will contain the following:
 The site is currently only ready to be run on a developer's machine.  As it
 becomes ready for staging, there will need to be instructions added for the
 various cloud instances running.  For now, as a developer, one can get it
-running on their machine with the following instructions in a command
-window:
+running on their machine with the following instructions in a command window
+(make sure to fork [this repository on
+GitHub](https://github.com/cognoma/django-cognoma "cognoma/django-cognoma on
+GitHub") first):
 
-    > git clone git@github.com:cognoma/django-cognoma.git
-    > cd django-cognoma
-    > virtualenv env
-    > source env/bin/activate
-    > pip install -r requirements.txt
-    > python manage.py migrate
-    > python manage.py createsuperuser
-    > python manage.py runserver
+```sh
+USERNAME=your_github_handle # Change to your GitHub Handle
```

Comment:
Yeah these instructions are targeted at developers. The current instructions are problematic because we want people to submit pull requests from their personal fork, since most people don't have write to `cognoma/django-cognoma`.

In short, the people cloning this repo ATM are most likely developers and not deployers.